test-cases/terraform/aws/iam/iam-entities/policy-too-broad/results_summary.json

```diff
@@ -1,7 +1,7 @@
 {
   "checkov": "no",
   "cloudrail": "yes",
-  "kics": "no",
+  "kics": "yes",
```

Comment:
@kaplanlior I don't see this being caught above?

Reply:
KICS says: IAM Policy Grants Full Permissions due to *
Cloudtrail says: Using wildcards (*) in the action and the resource can be overly permissive

Reply:
🤦 I missed that, it was in the section of the file that wasn't changed.